src/js/components/agencyV2/visualizations/ObligationsByAwardType.jsx

```diff
@@ -86,52 +109,84 @@ export default function ObligationsByAwardType({
+
             // Use the faded color when another section is hovered over
-            ((activeType && !categoryMapping[outer[i].label[0]].includes(activeType)) && !isMobile)
-                ? outer[i].fadedColor : outer[i].color)
-        )
+            if (activeType && !isMobile && activeCategory !== currentCategory) {
+                return outer[currentCategoryId].fadedColor;
+            }
+            return outer[currentCategoryId].color;
+        })
+        .style('cursor', 'pointer')
+        .on('mouseenter', (d) => {
+            // store the award type of the section the user is hovering over
+            setActiveType(d.data.label);
+            setCategoryHover(mapToFullCategoryName(d.data.type));
+        })
+        .on('mouseleave', () => {
+            setActiveType(null);
+            setCategoryHover(null);
+        })
         .attr('aria-label', (d) => `${d.data.label}: ${d3.format("($,.2f")(d.value)}`)
         .attr('role', 'listitem');
 
+    // white ring
+    chart.selectAll()
+        .data(pie)
+        .enter()
+        .append('path')
+        .attr('d', d3.arc()
+            .outerRadius(outerRadius - outerStrokeWidth)
+            .innerRadius(innerRadius)
+        )
+        .attr('fill', 'white')
+        .style('cursor', 'pointer')
+        .on('mouseenter', (d) => {
+            // store the award type of the section the user is hovering over
+            setActiveType(d.data.label);
+        })
+        .on('mouseleave', () => {
+            setActiveType(null);
+        })
+        .attr('role', 'listitem');
+
+
     // inner ring
     chart.selectAll()
-        .data(innerPie)
+        .data(pie)
         .enter()
         .append('path')
         .attr('d', d3.arc()
             .outerRadius(innerRadius)
             .innerRadius(innerRadius / 2)
         )
-        .attr('fill', (d, i) => (
+        .attr('fill', (d, i) => {
+            if (categoryHover && categoryHover === mapToFullCategoryName(d.data.type) && !isMobile) {
+                return inner[i].color;
+            }
+
             // Use the faded color when another section is hovered over
-            ((activeType && activeType !== inner[i].label) && !isMobile)
-                ? inner[i].fadedColor : inner[i].color)
-        )
-        .attr('aria-label', (d) => `${d.data.label}: ${d3.format("($,.2f")(d.value)}`)
+            return ((activeType && activeType !== inner[i].label) && !isMobile) ? inner[i].fadedColor : inner[i].color;
+        })
+        .style('cursor', 'pointer')
         .on('mouseenter', (d) => {
             // store the award type of the section the user is hovering over
             setActiveType(d.data.label);
         })
         .on('mouseleave', () => setActiveType(null))
-        .attr('role', 'listitem');
+        .attr('aria-label', (d) => `${d.data.label}: ${d3.format("($,.2f")(d.value)}`)
+        .attr('role', 'listitem')
+        .attr('tabindex', 0);
```

Comment:
Thanks for making these focusable! Maybe we could display/hide the tooltip `onFocus`/`onBlur`?

Reply:
Yeah, I spent a little time with this and adding onBlur and onFocus is causing an issue with tabbing which is weird.  I think it would be a good idea to create a separate ticket for the keyboard accessibility here to figure out what's blocking tabbing.